lib/task/s3.js

```diff
@@ -30,9 +30,10 @@ const setFailedToPending = withContainer(({ s3, Blobs }) => async () => {
   console.log(`${count} blobs marked for re-uploading.`);
 });
 
-const uploadPending = withContainer(({ s3, Blobs }) => async () => {
+const uploadPending = withContainer(({ s3, Blobs }) => async (limit) => {
   assertEnabled(s3);
-  const count = await Blobs.s3CountByStatus('pending');
+
+  const count = limit || await Blobs.s3CountByStatus('pending');
```

Comment:
Only weird thing is if you give it a limit higher than your # of blobs, it'll say it's uploading the limit

I have 0 blobs to upload here:
```
 node lib/bin/s3.js upload-pending 34
Uploading 34 blobs...
```

Reply:
Good catch, thanks!